modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -712,6 +713,10 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i,
     cv::MediaFrame frame = ctx.inFrame(i);
     if (ctx.uu.preproc_engine_impl) {
         GAPI_LOG_DEBUG(nullptr, "Try to use preprocessing for decoded remote frame in remote ctx");
+
+        //TODO
+        frame.blobParams();
```

Comment:
Todo?

Reply:
Oh, looks like there is some exception will be thrown if there is a CPU adapter inside

Reply:
I think I will change this block a little bit